java/runtime/src/main/java/io/ray/runtime/task/LocalModeTaskSubmitter.java

```diff
@@ -460,4 +472,11 @@ private static JavaFunctionDescriptor getJavaFunctionDescriptor(TaskSpec taskSpe
     }
     return returnIds;
   }
+
+  /** Whether this is an concurrent actor. */
```

Comment:
```suggestion
  /** Whether this is a concurrent actor. */
```

Reply:
What's the relationship between the task spec and the concurrent actor?

Reply:
Just fetch the maxConcurrency field from actor_creation_task_spec. None if it's not a creation spec. (PS: Similar with the method `getActorId(spec)`)

Reply:
Can you update the comment in code?